tests/services/test_links.py

```diff
@@ -0,0 +1,65 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2020 CERN.
+# Copyright (C) 2020 Northwestern University.
+#
+# Invenio-Records-Resources is free software; you can redistribute it and/or
+# modify it under the terms of the MIT License; see LICENSE file for more
+# details.
+
+"""Test links."""
+
+import pytest
+from flask_principal import Identity, Need, UserNeed
+
+from invenio_records_resources.services.record import RecordService
+
+
+@pytest.fixture()
+def identity_simple():
+    """Simple identity fixture."""
+    i = Identity(1)
+    i.provides.add(UserNeed(1))
+    i.provides.add(Need(method='system_role', value='any_user'))
+    return i
+
+
+@pytest.fixture()
+def identity_no_need():
+    """Simple identity fixture."""
+    i = Identity(1)
+    # i.provides.add(UserNeed(1))
+    # i.provides.add(Need(method='system_role', value='any_user'))
+    return i
+
+
+def test_record_links(app, identity_simple, input_service_data, es):
```

Comment:
Moderate: Missing docstring? I noticed in my local things do not fail with `./run-tests` it does not pick this things, needs investigation (separatedly, shelved).

Reply:
Docstrings are not needed for tests (by our linters). Personally I only exceptionally put a docstring in tests if there are more details than can be conveyed by the test name.